projects/canopy/src/lib/promo-card/promo-card-list/promo-card-list-title/promo-card-list-title.component.html

```diff
@@ -0,0 +1,6 @@
+<lg-heading
+  [class]="'lg-font--expressive lg-promo-card-list-title__heading lg-font-size-5--strong'"
```

Comment:
I wonder if we need to add an expressive modifier to the `lg-heading` component (@owensgit)

Reply:
Is this what we did in the standard cards? just wondering if we can avoid hardcoding classes.

Reply:
The other way around would be to add a variant to the header (e.g. normal / expressive)

Reply:
> I wonder if we need to add an expressive modifier to the lg-heading component (@owensgit)

I was going to add a new modifier to the `lg-heading` as part of the Lyon work - e.g. `<lg-heading fontType="expressive">`, but then wanted to check in with the rest of the team as we already have a `[class]` input that allows you to add the modifier class directly along with the other ones. So it could be argued that a specified modifier is superfluous. @elenagarrone  @andy-polhill what do you think?

> Is this what we did in the standard cards? just wondering if we can avoid hardcoding classes.

Yes we did - in the standard cards we only allow the `headingLevel` to be changed:

```
<lg-heading [level]="headingLevel" class="lg-card-title__heading">
  <ng-content></ng-content>
</lg-heading>
```

We were also wondering about the hardcoding of classes - but thought it may be a bit of YAGI to make the font type customisable from `<lg-promo-card-list-title>`, and the original card title implementation swayed us :) 

However, we could add an input, e.g. `<lg-promo-card-list-title fontType="expressive">` which gets passed down the to the `lg-heading`, in a similar way to the heading level, and then also possibly add the `fontType` input to the `lg-heading` too.

Maybe we can have quick chat about this on Monday?

Reply:
Ah I see, I think the `class` property in lg-heading is a bit of an anomaly, it was quite an early approach. On the whole we try not to promote the adding of arbitrary classes, as it can promote people to take their own path and also create unexpected breaking changes. We could look at deprecating that property separate from this PR.

For now I think I'm ok with the classes. I have a slight niggle in that we could use the scss mixin approach instead of the hardcoded css classes, but we haven't hooked `expressive` into the mixins yet

```html
<lg-heading [level]="headingLevel" class="lg-hero-card-title__heading">
```

```scss
.lg-promo-card-list-title__heading {
  @include lg-font-size('5', 'strong');
}
```

Reply:
Good point about the mixin and hooking in the `expressive` font. I'll add a new issue for that and also a discussion for deprecating the `class` at some point.

Reply:
Also just created an issue for potentially deprecating the `class` input on the lg-heading: https://github.com/Legal-and-General/canopy/issues/225